openfisca_france/param/param.xml

```diff
@@ -2781,7 +2784,16 @@
         <VALUE deb="2010-01-01" fin="2011-12-31" valeur="878" />
         <VALUE deb="2012-01-01" fin="2012-12-31" valeur="960" />
         <VALUE deb="2013-01-01" fin="2013-12-31" valeur="1016" />
-        <VALUE deb="2014-01-01" fin="2014-12-31" valeur="1021" />
```

Comment:
Adrien,

Célibataire, 1AJ =18.000 (revenus d’activités) et 1AK = 3.000 (frais réels)
-> Salaire imposable = 15.000

Patrick BODET

De :  adrienpacifico <notifications@github.com>
Répondre à :  openfisca/openfisca-france
<reply+001b32b238c70f86adc012b9ceee7c9961ece61e67e140d192cf0000000112198aa89
2a170ce026498b4@reply.github.com>
Date :  mardi 22 septembre 2015 23:31
À :  openfisca/openfisca-france <openfisca-france@noreply.github.com>
Cc :  Patrick BODET <patrick.bodet4@wanadoo.fr>
Objet :  Re: [openfisca-france] Decote 2015 (#324)


In openfisca_france/param/param.xml
<https://github.com/openfisca/openfisca-france/pull/324#discussion_r40147124
> :
> @@ -2781,7 +2784,16 @@
>          <VALUE deb="2010-01-01" fin="2011-12-31" valeur="878" />
>          <VALUE deb="2012-01-01" fin="2012-12-31" valeur="960" />
>          <VALUE deb="2013-01-01" fin="2013-12-31" valeur="1016" />
> -        <VALUE deb="2014-01-01" fin="2014-12-31" valeur="1021" />
Nope added a line before, and suppress a line 600 lines after.

—
Reply to this email directly or view it on GitHub
<https://github.com/openfisca/openfisca-france/pull/324/files#r40147124> .